### ISSUE TITLE
Have `View()` function that provides a more informative error

### ISSUE THREAD

**Issue by ismayc** [moderndive/moderndive#99]
I've talked to a couple of professors that are still struggling with students adding in `View()` to R Markdown documents and getting errors that are hard for the students to decipher. One way to "fix" this would be to include a wrapper function in this package that provides a more informative error if the environment being used was not interactive (like R Markdown). Otherwise, it behaves just like `View()` does usually.

Thoughts welcome.

**Comment by rudeboybert** [comment]
While I agree this would be nice to have, IMO the juice isn't worth the squeeze of figuring out how to program it

**Comment by ismayc** [comment]
Fair enough. I'll try to look into how this might be programmed to see how difficult it might be to check for reactivity or not.

**Comment by ismayc** [comment]
@rudeboybert This appears to be quite simple to implement since the `interactive()` function is in the {base} package.
Unfortunately, when `View()` is called in RStudio it takes over `utils::View()` which makes defining a new function called `View()` a little tricky.

It looks like they are working on having something similar to this in the {tibble} package with the `view()` function, but I think I'd like to have a `DT::datatable()` show up for people knitting the file instead of what `view()` does in having no effect for non-interactive sessions.

To try this out, put the following code in a chunk in an R Markdown document:

```
View <- function(x, ...) {
  if (interactive()) {
    tibble::view(x, ...)
  } else {
  DT::datatable(x, ...)
  }
}

View(moderndive::pennies)
```
